tensorboard/backend/application_test.py

```diff
@@ -317,25 +318,29 @@ def testPluginEntryBadModulePath(self):
       server.get('/data/plugin_entry.html?name=mallory')
 
 
-class ApplicationBaseUrlTest(tb_test.TestCase):
+class StandardTensorBoardWSGIAppUrlTest(tb_test.TestCase):
   path_prefix = '/test'
   def setUp(self):
     plugins = [
-        FakePlugin(plugin_name='foo'),
-        FakePlugin(
+        FakePluginLoader(plugin_name='foo'),
```

Comment:
I don’t follow; why are these changes required?

Reply:
Answered offline but for posterity:

Previously, it was using TensorBoardWSGI which does not call `make_plugin_loader` as opposed to now, it uses `standard_tensorboard_wsgi` which calls `make_plugin_loader` through TensorBoardWSGIApp. `make_plugin_loader` requires plugins to be a class (not instance) of TBPlugin or an instance of TBPluginLoader.